client/src/main/com/sinch/sdk/domains/conversation/adapters/models/requests/ContactCreateRequestParametersImpl.java

```diff
@@ -0,0 +1,38 @@
+package com.sinch.sdk.domains.conversation.adapters.models.requests;
+
+import com.sinch.sdk.core.models.OptionalValue;
+import com.sinch.sdk.domains.conversation.adapters.models.BaseContactImpl;
+import com.sinch.sdk.domains.conversation.adapters.models.ChannelIdentityImpl;
+import com.sinch.sdk.domains.conversation.models.ChannelType;
+import com.sinch.sdk.domains.conversation.models.LanguageType;
+import com.sinch.sdk.domains.conversation.models.requests.ContactCreateRequestParameters;
+import java.util.Collection;
+
+public class ContactCreateRequestParametersImpl extends ContactCreateUpdateRequestParametersImpl
+    implements ContactCreateRequestParameters {
+
+  protected ContactCreateRequestParametersImpl(
```

Comment:
Yes; this is managed during serialization.
Because of Builder pattern usage , required or not parameters won't be highlighted here and delegating required validation at serialization level enable:
- to not duplicate business here in addition to serialization layer
- if field required or not status change, there is no change onto this class: serialization (generated source) will do
- enable to store part of a fields within an object without business logic; Until they are really consumed then BL will be applied (this is not for this dedicated object:; it is a general rule common to this repository)